src/v2/Apps/Fair/Routes/FairArtworks.tsx

```diff
@@ -15,16 +25,34 @@ interface FairArtworksFilterProps {
 const FairArtworksFilter: React.FC<FairArtworksFilterProps> = props => {
   const { match, relay, fair } = props
   const { filtered_artworks } = fair
+  const { user } = useSystemContext()
 
   const hasFilter = filtered_artworks && filtered_artworks.id
 
   // If there was an error fetching the filter,
   // we still want to render the rest of the page.
   if (!hasFilter) return null
 
+  const { counts } = filtered_artworks
+
+  const filters = (
```

Comment:
This would be maximally flexible if it were an actual component:
```
const Filters = () => {
  return (
    <Box pr={2}>
      {!!user && <FollowedArtistsFilter />}

      <MediumFilter />
      <PriceRangeFilter />
      <WaysToBuyFilter />
      <GalleryFilter />
      <SizeFilter />
      <TimePeriodFilter />
      <ColorFilter />
    </Box>
  )
}